### DISCUSSION TITLE
Airplay volume issue

### DISCUSSION THREAD

**Discussion by derekngibbs** [music-assistant/support#2443] [Q&A] [answered]
Is anyone having issues with the latest beta version and the volume not actually changing. I'm playing Spotify on an airplay player. The new volume slider is moving, but the actual volume is not changing. This only happens on the main page. If I select the music playing the volume works on that screen.

**Comment by OzGav** [comment]
In the player selection view yes. In the control bar no

**Reply by OzGav** [reply, marked as answer]
This will be fixed in the next update

**Comment by jonostanck** [comment]
Having the same issue.
